docs/components/Page.tsx

```diff
@@ -81,51 +81,66 @@ export function DocsPage({
         css={{
           gridArea: 'main',
           position: 'relative',
-          paddingBottom: 'var(--space-xxlarge)',
+          display: 'grid',
+          gridTemplateRows: '4.5rem calc(100vh - 4.5rem)',
         }}
       >
         <Header />
         <Wrapper
           css={mq({
+            borderTop: '1px solid var(--border)',
+            overflowY: 'auto',
             display: ['block', null, 'grid'],
-            marginTop: '2.5rem',
-            gridTemplateColumns: noRightNav
-              ? '15rem minmax(0, auto)'
-              : [
-                  '15rem minmax(0, auto)',
-                  null,
-                  null,
-                  '10rem minmax(0, auto) 10rem',
-                  '15rem minmax(0, auto) 15rem',
-                ],
+            marginTop: '0rem',
+            gridTemplateColumns: '15rem minmax(0, 1fr)',
+            gridTemplateRows: '1fr auto',
             gap: ['var(--space-medium)', null, null, 'var(--space-large)', 'var(--space-xlarge)'],
           })}
         >
           <Sidebar isUpdatesPage={isUpdatesPage} />
-
-          <main
-            id="skip-link-content"
-            tabIndex={0}
-            ref={contentRef}
-            className={noProse ? '' : 'prose'}
+          <div
+            id="content-and-toc"
+            css={mq({
+              gridColumn: '2 / 3',
+              gridRow: '1 / 2',
+              display: ['block', null, 'grid'],
+              gridTemplateColumns: noRightNav
+                ? 'minmax(0, 1fr)'
+                : ['minmax(0, 1fr)', null, null, 'minmax(0, 1fr) 10rem', 'minmax(0, 1fr) 15rem'],
+              gap: ['var(--space-medium)', null, null, 'var(--space-large)', 'var(--space-xlarge)'],
+            })}
           >
-            <Stack
-              orientation="horizontal"
-              block
-              css={{ justifyContent: 'space-between', alignItems: 'baseline' }}
+            <main
+              id="skip-link-content"
+              tabIndex={0}
+              ref={contentRef}
+              className={noProse ? '' : 'prose'}
+              css={{
+                paddingTop: '2rem',
+                // to prevent default grid item min-width: auto behaviour
+                // it makes pre tags looooooong
```

Comment:
is this comment in reference to `paddingTop`?

Reply:
Yeah nah. This comment should go away. Added this comment before I figured out that minmax(0, 1fr) on grid container is enough to contain grid items to viewport width. I previously used minWidth: 0 on grid items but missed removing the comment when I made the change. Will remove it now.

Reply:
done